app/dts/behaviors/reset.dtsi

```diff
@@ -16,7 +16,7 @@
 
 		bootloader: behavior_reset_dfu {
 			compatible = "zmk,behavior-reset";
-			label = "BOOTLOADER_RESET";
+			label = "BOOTLOAD";
```

Comment:
Why does this need to be changed?  I'm missing something.

Reply:
I think these strings are sent as part of the message to tell the peripheral to invoke a behavior?

Reply:
Correct, the call to the peripheral passes behavior label (the unambiguous identifier we have for these things), the two params, and the state.

These had to be shortened to fit in a normal MTU for BLE.